test/testdrive/webhook.td

```diff
@@ -435,3 +435,9 @@ SELECT id FROM mz_sources WHERE name = 'webhook_bytes';
 
 > SELECT COUNT(*) FROM mz_internal.mz_storage_shards WHERE object_id = '${webhook-source-id}';
 0
+
+# Cleanup.
+> DROP CLUSTER webhook_cluster CASCADE;
+
+> DROP CLUSTER webhook_compute CASCADE;
+# Test cases for invalid characters in webhook names
```

Comment:
It'd be good to add a few test cases here that validate that `.` and `..` are not allowed in webhook source names, but that `foo.bar` is allowed. Also would be good to add a smattering of tests for idents in other places, e.g. `CREATE DATABASE "."` and `SELECT 1 AS "."`.

@MaterializeInc/testing can help with this!

Reply:
I will support on this!

Reply:
@matthewarthur: Is this already ready to test? I get some unexpected results but I am also not fully sure about the exact specification. Shall we maybe have a quick call about this topic?

Reply:
Added some tests with 6b29bce1bdcc404500833623c1b4fdc8066a067c as discussed with @matthewarthur.

Reply:
Should be fixed now!

Reply:
Excellent, thank you!